src/pages/components/FooterSection.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import DownloadButton from "./DownloadButton";
+
+const FooterSection = () => {
+  return (
+    <footer>
+      <section className="bg-gradient-to-t from-orange-900 via-orange-600 to-orange-500 h-[420px] lg:h-[420px]">
+        <div className="backgroundImagetyle">
+          <div>
+            <p className="leading-[65px] text-center text-5xl text-white font-bold font-nunito">
+              Your <span className="text-yellow-900">bestfriend</span> is <br />
+              waiting for <span className="text-yellow-900">you</span>
+            </p>
+          </div>
+          <div className="flex justify-center">
+            <DownloadButton
+              src="./images/GooglePlay.svg"
+              alt="google play store"
+              upperText="ANDROID APP ON"
+              lowerText="Google Play"
+              link="https://play.google.com/store"
+            />
+            <DownloadButton
+              src="./images/AppStore.svg"
+              alt="apple app store"
+              upperText="DOWNLOAD ON THE"
+              lowerText="App Store"
+              link="https://www.apple.com/ph/app-store/"
+            />
+          </div>
+        </div>
+      </section>
+
+      <section className="flex flex-col px-[20px] sm:px-[20px] md:px-[55px] lg:px-[200px]">
+        <div className="flex flex-col justify-between items-center md:flex-row">
+          <div className="flex flex-col justify-center items-center w-80 pb-6 md:pb-12 mt-7">
+            <img src="/images/LogoDarkFit.svg" alt="LogoDark" />
+            <p className="text-center md:text-left mt-8">
+              Swipet is a mobile application that makes pet adoption fun and
+              easier.
+            </p>
+          </div>
+          <div className="flex py-8 flex-col justify-center items-center font-nunito text-text-header-dark-color md:items-start">
+            <div className="font-bold text-2xl">Links</div>
+            <a href="/FAQs" className="mt-7">
+              FAQs
+            </a>
```

Comment:
- For links can we use the <Link> component provided by react router
- And also put hover state to the links you can use darker colors provided in the tailwind config